tech.keyforge.master.metainfo.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>tech.keyforge.master</id>
+  <developer id="tld.vendor">
+    <name>KeyForge</name>
+  </developer>
+  <name>KeyForge Master</name>
+  <summary>Configurator software for KeyForge devices</summary>
+  <url type="homepage">https://keyforge.tech/</url>
+  <url type="help">https://keyforge.tech/setup</url>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://keyforge.tech/end-user-agreement/</project_license>
```

Comment:
is the binary redistributable? If there is no explicit permission, `extra-data` shall be used.

Reply:
I need to do some research on it. I will make the necessary changes tomorrow.

Reply:
> is the binary redistributable? If there is no explicit permission, `extra-data` shall be used.

Yes, the binary is redistributable. I have updated the EULA page accordingly.

Reply:
> Redistribution: You may redistribute the binary form of the Software, unmodified, to any third party; however, you are prohibited from charging any fees for such redistribution or using such redistribution for any commercial gain.